### PR TITLE
mention JetBrains support

### PR DESCRIPTION
**Pull request by radoering**
JetBrains is supporting us with licenses for some years now and asked whether we would feature their logo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -113,3 +113,9 @@ external formats like virtual environments
 * [install.python-poetry.org](https://github.com/python-poetry/install.python-poetry.org): The official Poetry
 installation script
 * [website](https://github.com/python-poetry/website): The official Poetry website and blog
+
+## Sponsors
+
+Thanks to [JetBrains](https://www.jetbrains.com) for supporting us with licenses for their tools.
+
+[<img src="https://resources.jetbrains.com/storage/products/company/brand/logos/jetbrains.svg" width="150" alt="JetBrains logo." />](https://www.jetbrains.com)
```
Comment (radoering) [inline README.md:121]: Please check the size of the logo at https://github.com/radoering/poetry/tree/jetbrains-support?tab=readme-ov-file#sponsors

Adequate? Larger? Smaller?

File: README.md [hunk 2]
```diff
@@ -113,3 +113,9 @@ external formats like virtual environments
 * [install.python-poetry.org](https://github.com/python-poetry/install.python-poetry.org): The official Poetry
 installation script
 * [website](https://github.com/python-poetry/website): The official Poetry website and blog
+
+## Sponsors
```
[the remark this answers is not in the document]
Comment (abn) [reply README.md:117]: I would say "Supporters". Don't want to give people the idea they are funding development.

File: README.md (hunk 1 above)
Comment (abn) [reply README.md:121]: Logo size looks good. Should we consider a table instead?

Comment (radoering) [reply README.md:121]: Good idea, but if I do not miss anything, you cannot get rid of the header row and borders:

| | |
|---|---|
|[JetBrains logo.](https://www.jetbrains.com) | Thanks to [JetBrains](https://www.jetbrains.com) for supporting us with licenses for their tools.|

**Review by abn — APPROVED** [review]

**Pull request merged**